cross/sudo/PLIST

```diff
@@ -0,0 +1,36 @@
+bin:bin/sudo
+bin:bin/sudoedit
+bin:bin/sudoreplay
+bin:include/sudo_plugin.h
+bin:libexec/sudo/group_file.la
```

Comment:
Done, I did'nt find an extensive description of PLIST  does someone have a pointer ?